docs/synthetics/ci-cd-integration.md

```diff
@@ -122,6 +122,631 @@ AllLocations    276 us-west-1   passed  3/3 Passed  https://apps.sematext.com/ui
+    "scriptBased": true,
+    "conditions": [
+        {
+            "id": 1,
+            "type": "ERROR",
+            "operator": "=",
+            "value": "",
+            "enabled": true
+        },
+        {
+            "id": 2,
+            "type": "METRIC",
+            "key": "synthetics.time.response",
+            "operator": "<",
+            "value": "20000",
+            "enabled": true
+        }
+    ],
+    "alertRule": {
+        "schedule": [
+            {
+                "day": "Monday",
+                "index": 2,
+                "label": "MON",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Tuesday",
+                "index": 3,
+                "label": "TUE",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Wednesday",
+                "index": 4,
+                "label": "WED",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Thursday",
+                "index": 5,
+                "label": "THU",
+                "intervals": [
+                  {
+                     "start": "12:00",
+                     "end": "13:00"
+                   },
+                  {
+                     "start": "12:00",
+                     "end": "13:00"
+                   },
+                ]
+                "type": "CUSTOM"
+            },
+            {
+                "day": "Friday",
+                "index": 6,
+                "label": "FRI",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Saturday",
+                "index": 7,
+                "label": "SAT",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Sunday",
+                "index": 1,
+                "label": "SUN",
+                "intervals": [],
+                "type": "ACTIVE"
+            }
+        ],
+        "minDelayBetweenNotificationsInMinutes": "10",
+        "backToNormalNeeded": true,
+        "failedRunCountToAlert": 1,
+        "notificationsEnabled": true,
+        "useOnlyAlertRuleIntegrations": false
+    }
+}'
+```
+
+To create a Browser Monitor to monitor a website, we would send a HTTP request as follows:
+```
+curl -L -X POST 'https://apps.sematext.com/synthetics-api/api/apps/17174/monitors/browser' \
+-H 'Authorization: apiKey 9bddb0a6-xxxx-xxxx-xxxx-397d15806cfd' \
+-H 'Content-Type: application/json' \
+--data-raw '{
+    "name": "Example browser monitor without script",
+    "interval": "10m",
+    "enabled": true,
+    "locations": [
+        1,
+        2
+    ],
+    "url": "https://www.google.com/",
+    "script": "// Doesn'\''t matter since it doesn'\''t use a script",
```

Comment:
I'd just leave this field blank - `"script": ""`. I wrote it this way just to showcase that it wasn't needed for this variant of Browser monitor.